src/web/sitemap.rs

```diff
@@ -154,6 +154,7 @@ about_page!(AboutPageMetadata, "core/about/metadata.html");
 about_page!(AboutPageRedirection, "core/about/redirections.html");
 about_page!(AboutPageDownload, "core/about/download.html");
 
+#[axum::debug_handler]
 pub(crate) async fn about_handler(subpage: Option<Path<String>>) -> AxumResult<impl IntoResponse> {
```

Comment:
out of curiosity: wouldn't it be more straight-forward to have the subpages be part of the router and have dedicated handlers for them? :)

Reply:
probably :) 

this is historically coming the old iron routes, and I didn't change how they were structured when migrating to axum. 

( the list of possible refactors is very long 😄 , I'm trying to make progress as much as possible )

Reply:
I see :)